### ISSUE TITLE
Swipe gestures cannot be disabled on mobile

### ISSUE THREAD

**Issue by DrWhoCares** [appu1232/guyamoe#87]
Swipe gestures cannot be disabled and have no settings corresponding to them.
This makes the reader entirely unusable on my phone, since the entire reader becomes jittery and the slightest touch of the screen will cause it to switch pages, volumes, chapters, etc. The only time it's bearable is when dragging by first clicking on the top bar somewhere.

Having swipe gestures disabled would then mean that the image is entirely fixed, and no amount of sliding your finger across the screen will do anything except scroll up or down, left or right if zoomed.
Pages would be changed by enabling the tap to switch page option, or by using the top bar's controls.

**Comment by Algoinde** [comment]
It's a problem with your sensor sending ghost touches, correct?
But yeah, it makes a lot of sense to add that option. Thanks for reporting.

**Comment by DrWhoCares** [comment]
It probably is something to do with my phone, or the fact that I'm using Firefox 68, but I don't really have this issue anywhere else.
Regardless, it'll be fixed if the option is implemented. Thanks.

**Comment by Algoinde** [comment]
Ok, I think I know what this is. Does it only happen if you zoom in on a page?

**Comment by DrWhoCares** [comment]
No, unfortunately it happens literally always. The only time it does not is when I initiate a drag from the top bar, aka, not first touching the image in any way.

It seems like any time I touch the image, regardless of where, it tries to slide the image, and the sensitivity is so high that the slightest fluctuation left or right will move the image left or right, letting go in any position that is not the original X position will go either forward or back a page.

**Comment by DrWhoCares** [comment]
Just for extra information that I should've added sooner.
Samsung S20 on latest system update, Feb, (meaning I have the atrocious new update to OneUI).

**Comment by Algoinde** [comment]
Ok, that's way more interesting. Is it possible for you to capture a video of it happening?

**Comment by DrWhoCares** [comment]
I'd have to see if there's some built-in method of capturing the screen, otherwise I need some app to do so.
If you know of any, let me know and I can look into it. Currently the proxy reader isn't loading at all for me, but I'll give it a little bit, or close my browser to try to fix it if it keeps not loading.

**Comment by Algoinde** [comment]
You can try with any of the [Kaguya chapters](https://guya.moe/1).

[Screen recording guide](https://www.samsung.com/global/galaxy/what-is/screen-recorder/#:~:text=Swipe%20down%20at%20the%20top,done%2C%20you're%20recording!).

**Comment by DrWhoCares** [comment]
Did you change anything else other than adding the feature to turn it off? It actually feels a lot less bad than yesterday.
GitHub won't let me upload, so the video has to be small enough for me to host it on discord.
Instead of taking a bunch of videos trying to get it down to under 8mb to upload on discord. Here's two gifs:

https://cdn.discordapp.com/attachments/628096041580888064/817079932064104488/Screen_Recording_20210304-105851_Firefox_1.gif

https://cdn.discordapp.com/attachments/628096041580888064/817080665416663050/Screen_Recording_20210304-100710_Firefox_1.gif

Here's an example of with it off, notice it still a bit jittery, but that's not as important, since I wouldn't be dragging very often, after getting the image on screen properly.
[Video with Swipe OFF](https://cdn.discordapp.com/attachments/628096041580888064/817077596449538128/Screen_Recording_20210304-105043_Firefox.mp4)

**Comment by Algoinde** [comment]
Honestly I am having trouble understanding what exactly is jittery, because the gif framerate is so low. [Hop onto the kaguya discord](https://discord.gg/kaguya) and send the full vids in #tech-science, we have 100mb limit there.

**Comment by 68595959** [comment]
...
